src/main/groovy/org/labkey/gradle/task/RunTestSuite.groovy

```diff
@@ -33,6 +34,7 @@ class RunTestSuite extends RunUiTest
         project.logger.info("RunTestSuite: constructor");
         scanForTestClasses = false
         include "org/labkey/test/Runner.class"
+        // TODO: Remove when 19.2 is no longer supported. Dependency is now declared in `qc/build.gradle`
         if (project.findProject(":sampledata:qc") != null)
             dependsOn(project.project(":sampledata:qc").tasks.jar)
```

Comment:
I *think* it will be ok to have that dependency in the qc/build.gradle directory (since s comes before t for the current location and the qc project will be under the test project in the new location), but beware that this kind of dependency can cause order-of-operations problems.  If the project where the dependency is declared is evaluated before the project it depends on, the dependency won't get established.

Reply:
I ran into that problem with the `medImmune/build.gradle`, which adds a `dependsOn` to all of the test tasks. I changed it to use lazy task configuration and the problem cleared up. I used the same code in `qc/build.gradle` and it worked as expected